package.xml

```diff
@@ -33,6 +33,8 @@
   <test_depend>ament_cmake_copyright</test_depend>
   <test_depend>ament_cmake_uncrustify</test_depend>
   <test_depend>ament_cmake_lint_cmake</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
```

Comment:
remove `ament_cmake_lint_cmake`, `ament_cmake_cpplint, `ament_cmake_copyright`, `ament_cmake_uncrustify`

Reply:
As per the second section here: https://github.com/ament/ament_lint/blob/rolling/ament_lint_auto/doc/index.rst

Is it not correct to keep them in the test_dependencies or am I reading the documentation wrong?

Reply:
> [](https://github.com/ament/ament_lint/blob/rolling/ament_lint_auto/doc/index.rst)
> 
> Is it not correct to keep them in the test_dependencies or am I reading the documentation wrong?

They are unnecessary in this case because `ament_lint_common` brings them all in indirectly.

Reply:
Makes sense now, I'll have it updated in about 4 hours.